internal/core/attribute.cpp

```diff
@@ -236,7 +236,7 @@ void calc_attributes(
     auto fill = horizon.fillvalue();
 
     for (std::size_t i = from; i < to; ++i) {
-        auto depth = surface.at(i);
```

Comment:
For reference: the name `at` is taken from the standard library containers (vector, array). They define `at` as "access specified element with bounds checking".

Reply:
Yes, you said so. And because our structure is not a simple array, I always was confused about what it would return. Like does .at(i) return me a value of the cell or a coordinate of the cell?
I still consider RegularSurface to be a somewhat weird structure, but I think for now "value" fits better than "at".